client/main/notification.css

```diff
@@ -0,0 +1,76 @@
+
+.silky-notification {
+    z-index: 21 ;
+    
+}
+.silky-notification-box {
+    pointer-events: none ;
+    position: fixed ;
+    right: 10px ;
+    bottom: 10px ;
+    margin: 0 0 0 0 ;
+    padding: 5 15 5 15 ; 
+    background-color : lightgray ;
+    min-width: 500px ;
+    min-height: 50px ; 
+    box-shadow: 0 0 10px 5px gray ;
+}
+
+.silky-notification-content {
+    pointer-events: none ;
+    position: absolute ;
+    left: 0px ;
+    margin-right: 20 ;
+    padding: 12px ;
+    color: #1a1a1a ;
+    font-weight: 800 ;
+    font-size: little ;
```

Comment:
i don't think `little` is a thing:

https://developer.mozilla.org/en-US/docs/Web/CSS/font-size